tests/AdminCsvSpeakerFullNameFallbackTest.php

```
@@ -0,0 +1,178 @@
+    private function buildResourceServerContext(): IResourceServerContext
+    {
+        // TrackChairPresentationSerializer::serialize() passes this straight into
+        // Summit::getTrackChairByMember(Member $member), a non-nullable parameter, so this
+        // has to be a real Member mock rather than null even though none of these tests are
+        // about track-chair identity.
+        $current_user = Mockery::mock(Member::class);
+
+        $context = Mockery::mock(IResourceServerContext::class);
+        $context->shouldReceive('getApplicationType')->andReturn('JS_CLIENT');
+        $context->shouldReceive('getCurrentUser')->andReturn($current_user);
+        return $context;
```

PHPStan (2.2.8)

[error] Method Tests\AdminCsvSpeakerFullNameFallbackTest::buildResourceServerContext() should return models\oauth2\IResourceServerContext but returns Mockery\MockInterface.


(return.type)